src/query/QueryManager.jl

```diff
@@ -42,9 +58,13 @@ end
 
 function update(transaction::AbstractCoreTransaction, query::String, options = Proto.Options())
     db_result = stream(transaction, QueryManagerRequestBuilder.update_req(query, options))
-    answers = [entry.query_manager_res_part.update_res_part.answers for entry in db_result]
-    maps = map(ConceptMap, answers)
-    result = reduce(vcat, maps)
+    if db_result !== nothing
+        answers = [entry.query_manager_res_part.update_res_part.answers for entry in db_result]
+        maps = map(ConceptMap, answers)
```

Comment:
Same as previous function, can't `ConceptMap` be applied in the line above, instead of create a temporary vector?

Reply:
Same answer as previous.